cmd/artifact/command/push.go

```diff
@@ -47,8 +48,8 @@ func pushCommand(options *Options) *cobra.Command {
 			return nil
 		},
 	}
-	command.Flags().StringVar(&releaseManagerClient.BaseURL, "http-base-url", "https://release-manager.dev.lunarway.com", "address of the http release manager server")
-	command.Flags().StringVar(&releaseManagerClient.Metadata.AuthToken, "http-auth-token", "", "auth token for the http service")
+	command.Flags().StringVar(&releaseManagerClient.BaseURL, "http-base-url", os.Getenv("HAMCTL_URL"), "address of the http release manager server")
```

Comment:
Maybe we should use `ARTIFACT_*` instead as that's the name of the CLI?

Reply:
Yes, `HAMCTL` does not make that much sense. Either we use the cmd name, or just `SERVER_URL` or similar?

Reply:
If we change the existing prefix then we need a plan for how to migrate the environment variables.

Reply:
This CLI does not use env variables today, does it? It is solely flag based (and actually I think that is OK TBH. It is for CI purposes only.

Reply:
Makes sense. I confused the sub projects.